### ISSUE TITLE
Running python script in Rstudio (raven)

### ISSUE THREAD

**Issue by sr320** [RobertsLab/resources#1900]
TLDR - how do I run prepDE.py?

---

I am following the manual excerpt below...

in a python chunk 
ran

`/home/shared/stringtie-2.2.1.Linux_x86_64/prepDE.py`

console:
```
> reticulate::repl_python()
Would you like to create a default Python environment for the reticulate package? (Yes/no/cancel) Yes
Error in stop_no_virtualenv_starter(version = version, python = python) : 
  Suitable Python installation for creating a venv not found.
  Requested Python: /usr/bin/python3.6
Please install Python with one of following methods:
- https://github.com/rstudio/python-builds/
- reticulate::install_python(version = '<version>')
- sudo apt install python3-venv
```

Protocol: Using StringTie with DESeq2
Given a list of GTFs, which were re-estimated upon merging, users can follow the below protocol to use DESeq2 for differential expression analysis. To generate GTFs from raw reads follow the [StringTie protocol paper](http://www.nature.com/nprot/journal/v11/n9/full/nprot.2016.095.html) (up to the Ballgown step).
As described above, prepDE.py either accepts a .txt ([sample_lst.txt](https://ccb.jhu.edu/software/stringtie/dl/sample_lst.txt)) file listing the sample IDs and the GTFs' paths or by default expects a ballgown directory produced by StringTie run with the -B option. The following steps leads us through generating count matrices for genes and transcripts, importing this data into DESeq2, and conducting some basic analysis.

Generate count matrices using prepDE.py
Assuming default ballgown directory structure produced by StringTie 
`$ python prepDE.py`
List of GTFs sample IDs and paths ([sample_lst.txt](https://ccb.jhu.edu/software/stringtie/dl/sample_lst.txt))
`$ python prepDE.py -i sample_lst.txt`

**Comment by kubu4** [comment]
If you need python3, then use the following:

`python3 /home/shared/stringtie-2.2.1.Linux_x86_64/prepDE.py -i sample_lst.txt`
